app/backend/templates/donation_received.md

```diff
@@ -0,0 +1,25 @@
+---
+subject: "Thank you for your donation to Couchers.org!"
+---
+
+{% from "macros.html" import button, link, support_email %}
+
+Hi {{ user.name|couchers_escape }},
+
+Thank you for your donation of $ {{ amount }} to the Couchers.org Foundation. Your donation makes our work and the wellbeing of the community possible, and the project wouldn't exist without the support of you and other financial contributors!
+
+You can download an invoice and receipt for the donation here:
+
+{% if html %}
+
+{{ button("Invoice", receipt_url)|couchers_safe }}
+
+Alternatively, click the following link: {{ link(receipt_url, html)|couchers_safe }}.
+
+{% else %}
+<{{ receipt_url|couchers_escape }}>
+{% endif %}
```

Comment:
Is this URL permanent? And do they need to save the receipt or the URL? Whatever the answer, it should be in the email.

If they can view the invoice later from their account, that should be stated here as well.

Alternatively, it would be great if we could just attach the invoice for their records. Maybe future issue

Reply:
It's a URL from stripe, presumably permanent.

We will eventually add something to see all receipts and so on in the frontend, but I think this is fine for now. Email is also a good record.